vmdk_plugin/plugin.go

```diff
@@ -26,18 +26,21 @@ package main
 
 import (
 	"fmt"
-	"path/filepath"
-	"sync"
-	"time"
-
 	log "github.com/Sirupsen/logrus"
 	"github.com/docker/go-plugins-helpers/volume"
 	"github.com/vmware/docker-volume-vsphere/vmdk_plugin/utils/fs"
 	"github.com/vmware/docker-volume-vsphere/vmdk_plugin/vmdkops"
+	"golang.org/x/exp/inotify"
+	"path/filepath"
+	"sync"
+	"time"
 )
 
 const (
-	mountRoot = "/mnt/vmdk" // VMDK block devices are mounted here
+	devWaitTimeout   = 1
```

Comment:
One of what :-) ? Pls add units to either bar name devNotifyTimeoutSec or just comments